recirq/fermi_hubbard/extra-requirements.txt

```diff
@@ -1 +1 @@
-openfermion>=1.2.0
\ No newline at end of file
+openfermion>=1.3.0
```

Comment:
is there an actual need to bump this? I've changed openfermion to also lower-bound its Cirq; so it's not necessary to bump minimum openfermion just to get the latest cirq

Reply:
I thought it needed this for TwoQubitGate, but it looks like this was in 1.2.0.  I am not sure why I was getting errors about TwoQubitGate then.  Maybe my virtualenv was messed up.